source/sass/blocks/advantages.scss

```
@@ -7,10 +7,49 @@
 }
 
 .advantages__item {
+  position: relative;
   padding-top: 31px;
   padding-left: 70px;
   padding-right: 20px;
   text-align: left;
+
+  &::before {
+    content: "";
+    position: absolute;
+  }
+}
+
+.advantages__item--housing::before {
+  width: 40px;
+  height: 40px;
+  top: 21px;
+  left: 21px;
+  background-image: url("../img/icon-house.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
```

Можно вставлять ссылку прям в правило `content`, в случае, когда псевдо-элемент это изображение. Так более емко и удобочитаемее
```suggestion
.advantages__item--housing::before {
  content: url("../img/icon-house.svg");
  position: absolute;
  top: 21px;
  left: 21px;
}
```